dist/releases/4.19/config.toml

```diff
@@ -473,49 +472,60 @@ files = ["/usr/bin/container-disk"]
 
 [[payload.virt-cdi-importer-rhel9-container.ignore]]
 error = "ErrGoNotCgoEnabled"
-files = [
-"/usr/bin/cdi-containerimage-server"
-]
+files = ["/usr/bin/cdi-containerimage-server"]
+
+[[payload.virt-cdi-importer-rhel9-container.ignore]]
```

Comment:
@dominikholler  @aglitke can you add a one-line comment explaining why each of these errors is OK for each of the containers?